scripts/cli

```diff
@@ -50,6 +50,11 @@ cmd_build_json2type () {
   set -e
 }
 
+cmd_release () {
+  [[ `git branch --show-current` != "main" ]] || fail 'please do not release from `main` branch'
```

Comment:
Maybe we could be even more restrictive:

```suggestion
  ! [[ `git branch --show-current` =~ ^release/v ]] || fail 'please release from a `release/vX.Y.Z` branch'
```

Reply:
hum, the idea here is more to avoid to push something to main since then we need to remove it and that is a bit tedious with protection rules.
I don't think there is a great value to check the naming of the branch for the release. If we want to enforce the naming, I'd be in favor of letting the command create the branch and probably perform some extra checks.
That could be an improvement for another day.

Reply:
Alright. I agree that having a command to automate a bit the release would be nice.